tests/integration/Rs/Json/PatchRemoveTest.php

```diff
@@ -14,7 +14,7 @@ public function removeNullProvider()
             array(
                 '{"a":{"b":null}}', // target document
                 '[ {"op":"remove", "path":"/a/b"} ]', // patch document
-                '{"a":[]}' // expected doument
+                '{"a":{}}' // expected document
```

Comment:
@raphaelstolt this expectation in the test is wrong from my view. if i remove `b` from the *object* `{"b":null}`, there shouldn't be an `array` after the removal ;-)